packages/x-charts/src/PieChart/PieArcPlot.tsx

```diff
@@ -118,6 +125,27 @@ function PieArcPlot(props: PieArcPlotProps) {
           {...slotProps?.pieArc}
         />
       ))}
+      {/* Render the focus indicator last, so it can align nicely over all arcs */}
+      {focusedItem && (
+        <Arc
+          key={focusedItem.dataIndex}
+          startAngle={focusedItem.startAngle}
+          endAngle={focusedItem.endAngle}
+          paddingAngle={focusedItem.paddingAngle}
+          innerRadius={focusedItem.innerRadius}
+          color={focusedItem.color}
+          outerRadius={focusedItem.outerRadius}
+          cornerRadius={focusedItem.cornerRadius}
+          skipAnimation
+          stroke={(theme.vars ?? theme).palette.text.primary}
+          id={`${id}-focused`}
```

Comment:
This ID is the series ID, don't we want to use the original series ID?

Interactions (e.g., tooltip) don't seem to be working when it's focused:

https://github.com/user-attachments/assets/37fe61db-61ab-4a4e-a159-05f72c9251fd